docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,109 @@
+   - On the left, you see the **template JSON editor**. Here, you define the actual template descriptor. The descriptor follows the [most recent element template schema](https://github.com/camunda/element-templates-json-schema). The editor is divided into two sections: a read-only section, containing the schema reference, the template name, the template id, and an editable section, where you can [define your template descriptor](/components/modeler/desktop-modeler/element-templates/defining-templates.md).
+   - On the right, you see the live **Visual Preview**. The live preview shows how the properties panel will look when you apply the template to an element. It automatically updates on every valid change, and reflects the latest valid state of the template. The preview allows you to interactively check your template before publishing, including its usability.
+   - In the upper left, you can **Upload an icon** for your template. You can upload any image file you want, however we recommend to use squared SVG graphics. The icons get rendered 18x18 pixels in the element on the modeling canvas, and 32x32 pixels in the properties panel.
+
+   On every valid change, the template is saved automatically. In case of a problem, the template is not saved, and you need to [fix it beforehand](#fixing-template-problems).
+
+   ![Template editor anatomy](img/connector-templates/create-connector-template-2.png)
+
+4. Once you've written your template, publish it by clicking **Publish**. You will be prompted to optionally enter a version name. This name appears in your milestone list.
+
+   ![Publishing a template](img/connector-templates/create-connector-template-3.png)
+
+5. After publishing, the template can be applied to elements of all models in the same project.
+
+   :::info Project scope
+   A Connector template is always associated with a project, hence only available in the project where you added it.
+   :::
+
+### JSON editor features
+
+The JSON editor is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/). The Monaco Editor is the editor that powers VS Code. As a result, the template editor supports many familiar features, such as auto-formatting, indentation support, code completion, and error highlighting.
+
+Read the [VS Code editor docs](https://code.visualstudio.com/docs/editor/editingevolved) for a full overview of features.
+
+### Versioning Connector templates
+
+You can version your Connector templates using [milestones](/components/modeler/web-modeler/milestones.md), similar to diagrams.
+
+If you publish a new version of a Connector template and the old version is already used in diagrams, the user can either update these diagram Connector templates to the most recent version, or maintain the older version when editing the respective diagram element (see updating [element templates](/components/modeler/desktop-modeler/element-templates/using-templates.md#updating-templates)).
```

Comment:
Minor tweak:
Update text from "If you publish a new version of a Connector template and the old version is already used in diagrams, the user can either update these diagram Connector templates to the most recent version, or maintain the older version when editing the respective diagram element (see updating [element templates](/components/modeler/desktop-modeler/element-templates/using-templates.md#updating-templates))."

to "If you publish a new version of a Connector template and an older version is being used in diagrams, the user has two options, they can either update their diagrams to use the most recent version of the Connector template, or they can continue using the older version of the Connector template in their diagrams. See updating [element templates](/components/modeler/desktop-modeler/element-templates/using-templates.md#updating-templates)."

Reply:
I will go for 

> If you publish a new version of a Connector template and an older version is already being used in diagrams, the user can either [update the diagram elements](/components/modeler/desktop-modeler/element-templates/using-templates.md#updating-templates) to use the most recent version of the Connector template, or they can continue using the older version of the Connector template in their diagrams.

since this covers Max's, Christina's and your suggestions.